kernel/kernel-api/src/test/scala/io/delta/kernel/internal/util/SchemaUtilsSuite.scala

```diff
@@ -310,6 +311,140 @@ class SchemaUtilsSuite extends AnyFunSuite {
     }
   }
 
+  test("Compute schema changes with added columns") {
+    val fieldMappingBefore = Map(
+      1L -> new StructField("id", IntegerType.INTEGER, true)).map { case (k, v) =>
```

Comment:
Why is this map needed? I think this should be fine do you see a compile issue?
```
Map(1L -> new StructField("id", IntegerType.INTEGER, true)).asJava
```

Reply:
Yeah so we need a java.util.map<java.lang.Long, StructField> to pass to the API

```
Map(1L -> new StructField("id", IntegerType.INTEGER, true)).asJava
```

This converts into a java.util.Map<Scala Long, StructField>. The scala long in a map key doesn't autobox into Java long so I needed to explicitly convert those.

Reply:
Does this work? Just an idea -- not a blocker. Whatever you think is cleaner

```scala
val fieldMappingBefore = Map(
  java.lang.Long.valueOf(1L) -> new StructField("id", IntegerType.INTEGER, true)
).asJava
```

Reply:
I think that's cleaner, we can put this in a helper like @vkorukanti suggested and reduce duplication